### ISSUE TITLE
RN 0.59.10 启动出错：Module `AccessibilityInfo` does not exist in the Haste module map

### ISSUE THREAD

**Issue by git-life-everywhere** [reactnativecn/react-native-website#105]
通过 yarn start 启动的时候发现异常：

Loading dependency graph, done.
 BUNDLE  [dev] ./index.js ░░░░░░░░░░░░░░░░ 0.0% (0/1)::1 - - [02/Apr/2020:13:34:24 +0000] "GET /index.bundle HTTP/1.1" 500 - "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36"
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `/Users/coolqi/GitHubCode/AwesomeProject/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.
  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.
    at ModuleResolver.resolveDependency (/Users/coolqi/GitHubCode/AwesomeProject/node_modules/metro/src/node-haste/DependencyGraph/ModuleResolution.js:183:15)
    at ResolutionRequest.resolveDependency (/Users/coolqi/GitHubCode/AwesomeProject/node_modules/metro/src/node-haste/DependencyGraph/ResolutionRequest.js:52:18)
    at DependencyGraph.resolveDependency (/Users/coolqi/GitHubCode/AwesomeProject/node_modules/metro/src/node-haste/DependencyGraph.js:283:16)
    at Object.resolve (/Users/coolqi/GitHubCode/AwesomeProject/node_modules/metro/src/lib/transformHelpers.js:261:42)
    at /Users/coolqi/GitHubCode/AwesomeProject/node_modules/metro/src/DeltaBundler/traverseDependencies.js:399:31
    at Array.map (<anonymous>)
    at resolveDependencies (/Users/coolqi/GitHubCode/AwesomeProject/node_modules/metro/src/DeltaBundler/traverseDependencies.js:396:18)
    at /Users/coolqi/GitHubCode/AwesomeProject/node_modules/metro/src/DeltaBundler/traverseDependencies.js:269:33
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/coolqi/GitHubCode/AwesomeProject/node_modules/metro/src/DeltaBundler/traverseDependencies.js:87:24)

**Comment by sunnylqm** [comment]
这个一般是由于有某个原生模块不能正常初始化

**Comment by git-life-everywhere** [comment]
> 这个一般是由于有某个原生模块不能正常初始化

哥，我该怎么处理？？？？

**Comment by sunnylqm** [comment]
基本上得检查你装了哪些第三方，一个个排查
